common/tests/integration_tests.rs

```diff
@@ -0,0 +1,108 @@
+use light_client::*;
+use merkle_tree::*;
+use reserved::ReservedState;
+use simperby_common::{verify::CommitSequenceVerifier, *};
+
+#[test]
+fn basic1() {
+    let n = 10;
+    let keys = (0..n)
+        .into_iter()
+        .map(|i| generate_keypair(format!("{}", i)))
+        .collect::<Vec<_>>();
+    let members = keys
+        .iter()
+        .enumerate()
+        .map(|(i, (public_key, _))| Member {
+            public_key: public_key.clone(),
+            name: format!("member-{}", i),
+            governance_voting_power: 1,
+            consensus_voting_power: 1,
+            governance_delegations: None,
+            consensus_delegations: None,
+        })
+        .collect::<Vec<_>>();
+    let genesis_header = BlockHeader {
+        author: PublicKey::zero(),
+        prev_block_finalization_proof: Vec::new(),
+        previous_hash: Hash256::zero(),
+        height: 0,
+        timestamp: 0,
+        commit_merkle_root: Hash256::zero(),
+        repository_merkle_root: Hash256::zero(),
+        validator_set: members
+            .iter()
+            .map(|member| (member.public_key.clone(), member.consensus_voting_power))
+            .collect::<Vec<_>>(),
+        version: "0.1.0".to_string(),
+    };
+    let genesis_info = GenesisInfo {
+        header: genesis_header.clone(),
+        genesis_proof: keys
+            .iter()
+            .map(|(_, secret_key)| TypedSignature::sign(&genesis_header, secret_key).unwrap())
```

Comment:
`secret_key` -> `private_key`?

because i  think we express as private key mostly.

Reply:
you're right. (This code was generated by Copilot btw)

Reply:
fixed